foo.js

```diff
@@ -0,0 +1,28 @@
+import Alt, { Store } from './'
+
+const alt = new Alt()
+
+const actions = alt.generateActions('foo')
+
+class MyStore extends Store {
+  constructor() {
+    super(alt, {
```

Comment:
Not super happy with this, but I can live with just forcing passing the flux instance into super. Not the config though, I'd love to be able to pull that from static